### DISCUSSION TITLE
Two separate Auths on one website?

### DISCUSSION THREAD

**Discussion by lyolkinn** [supabase/supabase#27610] [Questions]
I have a project based on your "**nextjs-user-management**" template

Can I duplicate the Auth realization for two types of users? Different auth pages, user pages, different profiles tables in database?
How do I have two separate user groups/roles on Supabase?

**Comment by GaryAustin1** [comment]
You can add roles to users via various methods but auth is limited to 1 set of users with an email/phone as a common identifier.  So you can't have a user with the same email have to accounts.   You could add roles to their 1 account and use the roles to make client or RLS decisions on what they access.

https://supabase.com/docs/guides/database/postgres/custom-claims-and-role-based-access-control-rbac?queryGroups=language&language=plpgsql#create-auth-hook-to-apply-user-role
